connector/src/main/resources/schema/wildfly-jca_6_1.xsd

```diff
@@ -239,9 +160,35 @@
         </xs:attribute>
     </xs:complexType>
 
+    <xs:element name="short-running-threads" type="thread-pool">
+        <xs:annotation>
+            <xs:documentation>
+                Thread pool for short running jobs.
+                Long running jobs are identified by the HintsContext.LONGRUNNING_HINT with a value of true.
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+    <xs:element name="long-running-threads" type="thread-pool">
+        <xs:annotation>
+            <xs:documentation>
+                Thread pool for long running jobs.
+                Long running jobs are identified by the HintsContext.LONGRUNNING_HINT with a value of true.
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+    <xs:element name="elytron-enabled" type="xs:boolean" default="true">
+        <xs:annotation>
+            <xs:documentation>
+                <![CDATA[[
+                Indicates that Elytron is responsible for security for this workmanager. Default is true
+                ]]>
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
```

Comment:
@michpetrov This means a document containing one or more of these as root elements is valid. It also means one of these would be a legal child of a 'profile' element in standalone/domain/host.xml. Which of course would fail to parse.

Reply:
@bstansberry I've reverted it, unfortunately this is the only way xsd lets you reuse elements